### PR TITLE
[DEVX-1440] Add saucectl docs for artifacts cleanup

### PR DESCRIPTION
**Pull request by tianfeng92**
Add docs for artifact cleanup method for saucectl.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/testrunner-toolkit/configuration/common-syntax.md [hunk 1]
```diff
@@ -315,6 +316,16 @@ artifacts:
 ```
 ---
 
+### `cleanup`
+<p><small>| OPTIONAL | BOOLEAN |</small></p>
+
+Specifies whether to remove all contents of artifacts directory.
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/common-syntax.md:322]: How does this work? If set to true, all content (folders and files) of the specified download directory are cleared before any new artifacts from the current test are downloaded?

Comment (alexplischke) [reply docs/testrunner-toolkit/configuration/common-syntax.md:322]: Correct 👍

Comment (tianfeng92) [reply docs/testrunner-toolkit/configuration/common-syntax.md:322]: yes. It will remove artifacts directory before starting this run

Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/common-syntax.md:322]:
```suggestion
When set to `true`, all contents of the specified download directory are cleared before any new artifacts from the current test are downloaded.
```

File: docs/testrunner-toolkit/configuration/cypress.md [hunk 2]
```diff
@@ -363,6 +364,16 @@ artifacts:
 ```
 ---
 
+### `cleanup`
+<p><small>| OPTIONAL | BOOLEAN |</small></p>
+
+Specifies whether to remove all contents of artifacts directory.
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/cypress.md:370]:
```suggestion
When set to `true`, all contents of the specified download directory are cleared before any new artifacts from the current test are downloaded.
```

File: docs/testrunner-toolkit/configuration/espresso.md [hunk 3]
```diff
@@ -236,6 +237,16 @@ artifacts:
 ```
 ---
 
+### `cleanup`
+<p><small>| OPTIONAL | BOOLEAN |</small></p>
+
+Specifies whether to remove all contents of artifacts directory.
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/espresso.md:243]:
```suggestion
When set to `true`, all contents of the specified download directory are cleared before any new artifacts from the current test are downloaded.
```

File: docs/testrunner-toolkit/configuration/playwright.md [hunk 4]
```diff
@@ -357,6 +358,17 @@ artifacts:
 ```
 ---
 
+### `cleanup`
+<p><small>| OPTIONAL | BOOLEAN |</small></p>
+
+Specifies whether to remove all contents of artifacts directory.
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/playwright.md:364]:
```suggestion
When set to `true`, all contents of the specified download directory are cleared before any new artifacts from the current test are downloaded.
```

File: docs/testrunner-toolkit/configuration/puppeteer.md [hunk 5]
```diff
@@ -290,6 +291,16 @@ artifacts:
 ```
 ---
 
+### `cleanup`
+<p><small>| OPTIONAL | BOOLEAN |</small></p>
+
+Specifies whether to remove all contents of artifacts directory.
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/puppeteer.md:297]:
```suggestion
When set to `true`, all contents of the specified download directory are cleared before any new artifacts from the current test are downloaded.
```

File: docs/testrunner-toolkit/configuration/testcafe.md [hunk 6]
```diff
@@ -358,6 +359,16 @@ artifacts:
 ```
 ---
 
+### `cleanup`
+<p><small>| OPTIONAL | BOOLEAN |</small></p>
+
+Specifies whether to remove all contents of artifacts directory.
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/testcafe.md:365]:
```suggestion
When set to `true`, all contents of the specified download directory are cleared before any new artifacts from the current test are downloaded.
```

File: docs/testrunner-toolkit/configuration/xcuitest.md [hunk 7]
```diff
@@ -243,6 +244,16 @@ artifacts:
 ```
 ---
 
+### `cleanup`
+<p><small>| OPTIONAL | BOOLEAN |</small></p>
+
+Specifies whether to remove all contents of artifacts directory.
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/xcuitest.md:250]:
```suggestion
When set to `true`, all contents of the specified download directory are cleared before any new artifacts from the current test are downloaded.
```

File: docs/testrunner-toolkit/saucectl.md [hunk 8]
```diff
@@ -284,6 +284,16 @@ saucectl run --artifacts.download.when always
 ```
 ---
 
+### `--artifacts.cleanup <boolean>`
+<p><small>| OPTIONAL | BOOLEAN |</small></p>
+
+Specifies whether to remove all contents of artifacts directory.
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/saucectl.md:290]:
```suggestion
When set to `true`, all contents of the specified download directory are cleared before any new artifacts from the current test are downloaded.
```

**Review by sweeneyskirt-sl — COMMENTED** [review]
I added suggestions for the more precise description -- you can accept them or, if you think it is overkill, just ignore. :)

**Review by sweeneyskirt-sl — APPROVED** [review]

**Comment by tianfeng92** [comment]
Looks great. Thanks @sweeneyskirt-sl

**Pull request merged**